packages/calypso-e2e/src/lib/components/editor-settings-sidebar-component.ts

```diff
@@ -270,8 +269,8 @@ export class EditorSettingsSidebarComponent {
 			if ( key === 'month' ) {
 				// For month numbers less than 10, pad the digit to be
 				// 2 digits as required by the select.
-				const monthSelectLocator = this.editor.locator( selectors.scheduleMonthSelect );
-				await monthSelectLocator.selectOption( date[ key ].toString().padStart( 2, '0' ) );
+				const monthSelectLocator = this.editor.locator( selectors.scheduleInput( 'month' ) );
+				await monthSelectLocator.selectOption( ( date[ key ] + 1 ).toString().padStart( 2, '0' ) );
```

Comment:
The list's values start with 01, and for some reason, `date['month']` was `00` for January, while in the list the value was `01`. Didn't dig too deep into this, but the '+1` fixed the issue.

Reply:
Weird, must have changed with this release? Anyway, as long as it's passing.